### PR TITLE
fix(attachment-carousel): css cleanup

### PR DESCRIPTION
**Pull request by braddialpad**
# fix(attachment-carousel): css cleanup

## :hammer_and_wrench: Type Of Change

These types will increment the version number on release:

- [x] Fix

## :book: Jira Ticket

https://dialpad.atlassian.net/browse/DLT-2122

## :book: Description

CSS recipe cleanup with the primary goal being to remove as much nested CSS as possible, and use proper BEM naming conventions (no double __ for example).

### CODE REVIEW

File: packages/dialtone-css/lib/build/less/recipes/attachment_carousel.less [hunk 1]
```diff
@@ -3,7 +3,7 @@
   width: var(--dt-space-1000);
   max-height: 100px; /* stylelint-disable-line meowtec/no-px */
 
-  .d-recipe-attachment-carousel__media-list {
+  &__media-list {
```
Comment (juliodialpad) [inline packages/dialtone-css/lib/build/less/recipes/attachment_carousel.less]: I was keeping them like `.d-recipe-attachment-carousel__media-list` as it's a lot simplier to find those classes when you're debugging or trying to fix something quickly.

Might seem too verbose, but it's a good practice I read somewhere and made sense for me.

Comment (juliodialpad) [reply packages/dialtone-css/lib/build/less/recipes/attachment_carousel.less]: Just need to move them to the right place of course

Comment (braddialpad) [reply packages/dialtone-css/lib/build/less/recipes/attachment_carousel.less]: I feel like this is one of those debated topics in CSS. writing the whole class name gives it better searchability, but using `&` makes it easier to see what is related.

I made the restructure, let me know if that's what you were thinking.

**Review by juliodialpad — APPROVED** [review]
Yes, this is more likely what I was looking for, thanks!

**Pull request merged**